src/StackExchange.Redis/ConfigurationOptions.cs

```diff
@@ -429,6 +432,15 @@ public int KeepAlive
             set => keepAlive = value;
         }
 
+        /// <summary>
+        /// The <see cref="ILogger"/> to use for connection events.
+        /// </summary>
+        public ILogger? Logger
```

Comment:
add `Logger = Logger,` to `Clone`

Reply:
Aye, need to add to DefaultOptions too, I think - was confirming with @philon-msft on design but I think 👍 to both it is

Reply:
Alrighty, added these in last push!